src/witan/gateway/queries/search.clj

```diff
@@ -47,8 +50,12 @@
 
 (defn datapack-files
   [u d search]
-  (minimal-file-search u d search))
+  (minimal-metadata-search u d
+                           (assoc search
+                                  :metadata-type "stored")))
 
 (defn datapack-files-expand
```

Comment:
These functions appear identical. Is there actually a difference? Also, the name of the fns is a bit confusing. Are these searching files within a datapack?